servers/murder/powerrounds/lua/autorun/config_powerrounds.lua

```diff
@@ -118,6 +118,28 @@ end
 
 		Other standalone useful functions can be found in sh_powerrounds.lua line 358
 ]]
+
+PowerRounds.AddRound({
+    Name = "Assassino invisível",
+    Gamemode = "Murder",
+    Description = "O assassino é invísivel, mas sua localização é revelada de 10 em 10 segundos. Todos inocentes têm a arma.",
+    PlayersStart = function(Ply)
+        if Ply:IsMurderer() then
+            Ply:SetRenderMode(RENDERMODE_TRANSALPHA)
+            Ply:SetColor( Color(0, 0, 0, 0 ) )
```

Comment:
Gmod has a built in transparent color, as you can see here: https://github.com/Facepunch/garrysmod/blob/cf262a1342ccb79831046b98452317ba1a1f9e3b/garrysmod/lua/includes/util.lua#L156

```suggestion
            Ply:SetColor(color_transparent)
```